apps/docs/content/guides/auth/auth-email-templates.mdx

```diff
@@ -4,21 +4,24 @@ description: 'Learn how to manage the email templates in Supabase.'
 subtitle: 'Learn how to manage the email templates in Supabase.'
 ---
 
-You can customize the email messages used for the authentication flows and security notifications. You can edit the following authentication-related email templates:
+You can customize the email messages used for the authentication flows and security notifications. The following authentication-related email templates can be configured:
 
-- Confirm signup
+- Confirm sign up
 - Invite user
-- Magic Link
-- Change Email Address
-- Reset Password
+- Magic link
+- Change email address
+- Reset password
+- Reauthentication
```

Comment:
This Reauthentication one was missing. Is this okay to add in @issuedat? See also the `mailer_subjects_reauthentication` and `mailer_templates_reauthentication_content` in the below table.

Reply:
We should probably add `{{ .Token }}` to the _Terminology_ table